spec/features/user_sees_all_dinosaurs_spec.rb

```diff
@@ -22,4 +23,11 @@
     expect(page).to have_content(dino.name)
   end
 
+  scenario 'dinosaur names are links to show pages' do
+    dino = FactoryGirl.create(:dinosaur)
+    visit root_path
+    click_link (dino.name)
+    current_path.should == dinosaur_path(dino)
```

Comment:
Use the RSpec `expect` syntax:

`expect(current_path).to eq(dinosaur_path(dino)`